rdmd_test.d

```diff
@@ -46,40 +50,55 @@ bool verbose = false;
 
 void main(string[] args)
 {
-    string buildCompiler = "dmd";  // e.g. dmd/gdmd/ldmd
-    string rdmd = "rdmd.d";
+    string rdmd; // path to rdmd executable
+    string defaultCompiler; // name of default compiler expected by rdmd
     bool concurrencyTest;
     string model = "64"; // build architecture for dmd
     string testCompilerList; // e.g. "ldmd2,gdmd" (comma-separated list of compiler names)
 
-    getopt(args,
-        "compiler", &buildCompiler,
-        "rdmd", &rdmd,
-        "concurrency", &concurrencyTest,
-        "m|model", &model,
-        "test-compilers", &testCompilerList,
-        "v|verbose", &verbose,
+    auto helpInfo = getopt(args,
+        "rdmd", "[REQUIRED] path to rdmd executable to test", &rdmd,
+        "rdmd-default-compiler", "[REQUIRED] default D compiler used by rdmd executable", &defaultCompiler,
```

Comment:
getopt supports `config.required`

Reply:
Fairly sure a previous version of this PR used it. I guess it was taken out to allow displaying a better error message?